src/core/executer.ts

```diff
@@ -50,7 +50,7 @@ export class Executer {
         process.on('exit', this.closeChild);
 
         child.on('exit', (code) => {
-            this.logger.info(`"${name}" PID ${child.pid} ${code === 0 ? 'done' : `exit ${code}`}`);
```

Comment:
The code may be null when the execution is successful, so change it to ```${!code ? 'done' : `exit ${code}`}```. Is this better?

Reply:
do

Reply:
done